### ISSUE TITLE
[2.8] Have a problem with debug:container

### ISSUE THREAD

**Issue by henres** [symfony/symfony#17007]
Can't find the title method in the ConsoleOutput class and in his extends/implements.
Can't reproduce the problem with standard project
```php
php app/console debug:container -vvv
[2015-12-14 14:57:16] event.DEBUG: Notified event "console.command" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure".  
[2015-12-14 14:57:16] event.DEBUG: Notified event "console.command" to listener "Symfony\Bridge\Monolog\Handler\ConsoleHandler::onCommand".  
PHP Fatal error:  Call to undefined method Symfony\Component\Console\Output\ConsoleOutput::title() in /var/www/come1job/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php on line 181
[2015-12-14 14:57:16] php.CRITICAL: Fatal Error: Call to undefined method Symfony\Component\Console\Output\ConsoleOutput::title() {"type":1,"file":"/var/www/come1job/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php","line":181,"level":-1,"stack":[]} 


                                                                                                                  
  [Symfony\Component\Debug\Exception\UndefinedMethodException]                                                    
  Attempted to call an undefined method named "title" of class "Symfony\Component\Console\Output\ConsoleOutput".  
                                                                                                                  


Exception trace:
 () at /var/www/come1job/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php:181


debug:container [--show-private] [--tag TAG] [--tags] [--parameter PARAMETER] [--parameters] [--format FORMAT] [--raw] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-s|--shell] [--process-isolation] [-e|--env ENV] [--no-debug] [--] <command> [<name>]
```

**Comment by stof** [comment]
@javiereguiluz can you check this ?

**Comment by javiereguiluz** [comment]
@stof sure! But @henres says that *"I can't reproduce the problem with standard project"*. So we'll need more information about your application to discover what's the origin of this error. Thanks.

**Comment by javiereguiluz** [comment]
I recently made some changes (https://github.com/symfony/symfony/commit/c658539d3ef3d299a0c9510020d73774a6f5d9b1) and I introduced some bugs. But they were very quickly fixed in a separate pull request (https://github.com/symfony/symfony/commit/e18a42af49e59440b3b61b32c35212e19ce07540).

**Comment by stof** [comment]
@henres please show the output of ``composer show -i symfony/symfony``

**Comment by xabbuh** [comment]
see #16955

**Comment by henres** [comment]
I have the latest version with your pullrequest that use io instead of output.
My composer still running, so i post the result of @stof command in some times

**Comment by stof** [comment]
I just merged the PR into 2.8

**Comment by henres** [comment]
Ok thanks guys, works like a charm now with the latest PR :). Can debug my own code now x)
